### PR TITLE
Exercise: Electronic Life Project

### PR DESCRIPTION
**Pull request by artem-konovalov-personal**
Closed [#396](https://github.com/nesterone/sf-eng-init-gl-spring-2016/issues/396)
Closed [#397](https://github.com/nesterone/sf-eng-init-gl-spring-2016/issues/397)
Closed [#398](https://github.com/nesterone/sf-eng-init-gl-spring-2016/issues/398)
Closed [#399](https://github.com/nesterone/sf-eng-init-gl-spring-2016/issues/399)
Closed [#400](https://github.com/nesterone/sf-eng-init-gl-spring-2016/issues/400)

### CODE REVIEW

File: 01-syntax/06-project-elife/elife.js [hunk 1]
```diff
@@ -1 +1,399 @@
...
+    }
+
+    if (space) {
+      return { type: 'move', direction: space };
+    }
+
+    return undefined;
+  };
+
+  window.valley = new LifelikeWorld(plan, {
```
Comment (nesterone) [inline 01-syntax/06-project-elife/elife.js]: I propose to rid off `valley` completely and initialize   `LifelikeWorld` directly in 

https://github.com/nervebassmaster/sf-eng-init-gl-spring-2016/blob/elifeProject/01-syntax/06-project-elife/index.html#L8

**Comment by artem-konovalov-personal** [comment]
Exercise: Replicate Elife Sceleton **Done**

**Comment by artem-konovalov-personal** [comment]
Closed [#397](https://github.com/nesterone/sf-eng-init-gl-spring-2016/issues/397)

**Comment by artem-konovalov-personal** [comment]
Closed [#398](https://github.com/nesterone/sf-eng-init-gl-spring-2016/issues/398)

**Comment by artem-konovalov-personal** [comment]
Closed [#399](https://github.com/nesterone/sf-eng-init-gl-spring-2016/issues/399)

**Comment by artem-konovalov-personal** [comment]
Closed [#400](https://github.com/nesterone/sf-eng-init-gl-spring-2016/issues/400)

File: 01-syntax/06-project-elife/elife.js [hunk 2]
```diff
@@ -1 +1,510 @@
...
+    '&': TastyPlant,
+    W: SwoopingPlant
+  });
+}
+
+window.onload = function () {
+  var areaSelector;
+
+  areaSelector = document.getElementsByTagName('select')[0];
+  areaSelector.onchange = function (event) {
```
Comment (nesterone) [inline 01-syntax/06-project-elife/elife.js:502]: I see, good addition

**Pull request merged**